### ISSUE TITLE
Include url to device wiki page in deviceinfo

### ISSUE THREAD

**Issue by PabloCastellano** [postmarketOS/pmbootstrap#434]
IMO, it would be useful to link each device page in the [devices](https://github.com/postmarketOS/pmbootstrap/wiki/Devices) page from their deviceinfo.

For example, for asus-grouper:

```
# Reference: <https://postmarketos.org/deviceinfo>
# Please use double quotes only. You can source this file in shell scripts.
# Wiki: <https://github.com/postmarketOS/pmbootstrap/wiki/asus-grouper-%28Google-Nexus-7-2012%29>
...
```

What do you think?
We can also beautify the urls with aliases like `postmarketos.org/devices/grouper`

**Comment by ollieparanoid** [comment]
That sounds like a good idea! But make sure not to introduce additional effort (the short URLs look very nice, but it sounds like some effort).

PS: If we would do such a short URL thing, I would recommend always including the manufacturer in the code name, otherwise we might have clashes at some point.

**Comment by drebrez** [comment]
@PabloCastellano @ollieparanoid with the new wiki, the urls are not that bad, they can be added without having to create additional short urls, or not?

**Comment by ollieparanoid** [comment]
Yep, they are much better and I agree that we could use them without short urls:
```
https://wiki.postmarketos.org/wiki/Google_Nexus_7_2012_(asus-grouper)
```

**Comment by MartijnBraam** [comment]
Short urls could be possible but the list for them has to be maintained seperately (in the build stuff for the postmarketos.org site or with a .htacces file on the wiki.postmarketos.org domain)

**Comment by ollieparanoid** [comment]
I would prefer if we did not do that, because that would involve an additional PR for the pmos.org page for every new device, which increases the entry barrier a bit (and means a bit more reviewing work etc).

**Comment by MartijnBraam** [comment]
Yeah definitely, the only way short urls could work good if we somehow generate the redirect list from wiki page contents (which we have in the other git repo) but that would be messy and complex for not a lot of benefit.

**Comment by Halamix2** [comment]
Since we require (or at least it is preferred) for each device to have its own wiki page maybe we could reuse `url` attribute of `device-<codename>` `APKBUILD`?
Currently it contains very generic `url="https://postmarketos.org"`
